### PR TITLE
WIP: fix: replace into clause because of the history triggers

### PR DESCRIPTION
**Pull request by tuncheto**
How can I test the creation of transaction? I couldn't find in the UI

### CODE REVIEW

**Comment by kalinkrustev** [comment]
We will hold this for a while, as transfer table is not currently subject of the generic audit logging due to performance concerns. We already have something similar to audit implemented in the transfer.event table.

File: api/sql/schema/750$transfer.push.create.sql [hunk 1]
```diff
@@ -105,6 +105,59 @@ BEGIN TRY
     WHERE
         partnerId = @ledgerId
 
+    
+    DECLARE @result TABLE(
```
Comment (kalinkrustev) [inline api/sql/schema/750$transfer.push.create.sql]: Can we
`DECLARE @result transfer.transferTT` 
and 
`OUTPUT INSERTED.* INTO @result`
and then
`SELECT *,
        @merchantMode,
        REPLACE(REPLACE(REPLACE(CONVERT(varchar, @merchantSettlementDate, 120),'-',''),':',''),' ',''),
        @merchantSerialNumber,
        @merchantSettings,
        @issuerMode,
        REPLACE(REPLACE(REPLACE(CONVERT(varchar, @issuerSettlementDate, 120),'-',''),':',''),' ',''),
        @issuerSerialNumber,
        @issuerSettings,
        @issuerPort,
        @ledgerPort,
        @ledgerMode,
        @ledgerSerialNumber
FROM @result
`

Comment (tuncheto) [inline api/sql/schema/750$transfer.push.create.sql]: it is not a good idea to use *

**Comment by zetxx** [comment]
what is going on here?
@kalinkrustev @tuncheto ?

**Comment by kalinkrustev** [comment]
We do not need to have history for the transfer table, as we already have transfer.event, which is supposed to do serve more advanced functionality.

**Pull request closed without merging**